src/components/PostPageDetail/index.js

```diff
@@ -678,7 +677,7 @@ const PostPageDetailIdComponent = (props) => {
   );
 };
 
-export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
+export default PostPageDetailIdComponent;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the Higher Order Component (HOC) `withInteractionsManaged` from the default export might have implications on how interactions are managed in this component. Ensure that this change does not affect the functionality of the component adversely. If `withInteractionsManaged` was providing important functionality such as managing user interactions, consider reimplementing those functionalities in the `PostPageDetailIdComponent` itself.


<!-- This is an auto-generated comment by OSS CodeRabbit -->